fairscale/optim/adascale.py

```diff
@@ -582,3 +587,47 @@ def set_num_gradients_to_accumulate(self, num_gradients_to_accumulate: int, upda
             # When effective world size is large enough, smoothing is probably
             # not needed, so the smoothing factor is 0.
             self._smoothing = max(1 - self._world_size * self._num_grads_to_accum / 1000, 0)
+
+
+class AdaScaleWrapper(AdaScale):
```

Comment:
should this be in an experimental folder since we are still validating this approach?

Reply:
only OSS(AdaScale) is experimental. The wrapper can totally be used for AdaScale(OSS), just a different way of initialization.